Library/Formula/pkcrack.rb

```diff
@@ -0,0 +1,185 @@
+class Pkcrack < Formula
+  homepage "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack.html"
+  url "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack/pkcrack-1.2.2.tar.gz"
+  version "1.2.2"
+  sha256 "4d2dc193ffa4342ac2ed3a6311fdf770ae6a0771226b3ef453dca8d03e43895a"
+
+  # Apply to the patch to build on the OSX
```

Comment:
Can you link to an upstream commit or bug report here? Otherwise it's a complete nightmare to keep track of long-term. If it hasn't been submitted upstream yet, It needs to be.